From akpm@osdl.org Thu Jan  5 22:54:28 2006
Message-Id: <200601060650.k066ooIj008824@shell0.pdx.osdl.net>
From: Andrew Morton <akpm@osdl.org>
Subject: USB: usbip more dead code fix
To: greg@kroah.com
Cc: akpm@osdl.org, taka-hir@is.naist.jp
From: akpm@osdl.org
Date: Thu, 05 Jan 2006 22:50:37 -0800


From: Andrew Morton <akpm@osdl.org>

drivers/usb/ip/vhci_hcd.c:639: warning: `vhci_device_unlink_all_urb' defined but not used

Cc: Takahiro Hirofuchi <taka-hir@is.naist.jp>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/usb/ip/vhci_hcd.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/ip/vhci_hcd.c
+++ gregkh-2.6/drivers/usb/ip/vhci_hcd.c
@@ -634,7 +634,7 @@ static int vhci_urb_dequeue(struct usb_h
 	return 0;
 }
 
-
+#if 0
 static void vhci_device_unlink_all_urb(struct vhci_device *vdev)
 {
 	struct vhci_priv *priv, *tmp;
@@ -674,6 +674,7 @@ static void vhci_device_unlink_all_urb(s
 
 	spin_unlock(&vdev->priv_lock);
 }
+#endif
 
 /*
  * The important thing is that only one context begins cleanup.
